include/Table.h

```diff
@@ -39,13 +39,14 @@ class RowBuilder;
  * Reprentation of result tables returns by VoltDB.
  */
 class Table {
-    friend class RowBuilder;
+    friend class TableTest;
```

Comment:
For testing purpose, table test needs access to bytebuffer maintained by table. So made it a friend. Another option could be provide accessor function to fetch byte buffer for testing but as did not think there is need for the user to see bytebuffer, so keeping table test as a friend thought is a feasible option.